src/software/ai/hl/stp/tactic/receiver_tactic.h

```diff
@@ -75,11 +76,11 @@ class ReceiverTactic : public Tactic
      * @param robot The receiver robot
      * @param ball The ball
      * @param best_shot_target The point the receiver robot should shoot at
-     * @return A pair containing the position and orientation the receiver robot should
-     * have in order to perform a one-time shot
+     * @return A Shot object containing the position and orientation the receiver robot
+     * should have in order to perform a one-time shot
```

Comment:
I'd suggest something like:
```
     * @return A Shot that the given robot can perform with the given ball in order to shoot at the given point
```
as the exact details of what a shot contains are part of the `Shot` class now